lib/challenge.js

```diff
@@ -51,7 +66,14 @@ var pet_owner;
 var some_pet;
 
 // Answer Starts Here
-
+some_pet = {
+  name: "Brownie",
+  species: "dog",
+  breed: "Border collie",
+  makeNoise: function(noise) {
+    console.log(noise);
```

Comment:
We were specifically looking for `makeNoise` to utilize the object's `noise` property; it should end up looking something like this:
```javascript
somePet = {
  name : "Brownie",
  species : "dog",
  breed : "Border Collie",
  noise : "woof"
  makeNoise : function(){
    console.log(this.noise);
  } 
};
```